libp2p/CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # 0.51.1
 
 - Depend on `libp2p-tls` `v0.1.0`.
+- Deprecate the `quic` and `webrtc` feature.
+  These two crates are only in alpha state.
+  To properly communicate this to users, we want them to add the dependency directly which makes the `alpha` version visible.
+  It is recommended to pin the versions of `libp2p-quic` and `libp2p-webrtc` using `=`.
```

Comment:
```suggestion
```

With https://github.com/libp2p/rust-libp2p/pull/3584 pinning should no longer be required, correct?

Reply:
Ah right, because we will bump the minor versions.